### ISSUE TITLE
Enable / Disable Cron not working

### ISSUE THREAD

**Issue by terry-sydaus** [raskitoma/pls_admin#4]
The following commands do not work:

```bash
docker exec -it adminpls enable_cron.sh
```

and

```bash
docker exec -it adminpls disable_cron.sh
```

I ran the following command instead to keep the db synced.

```bash
docker exec -it adminpls python3 -m flask scheduler
```

**Comment by raskitoma** [comment]
Fixed on commit #[9b51149](https://github.com/raskitoma/pls_admin/commit/9b5114982072e606e2cbf5e20727015c1ec1590f)

Instead of relying on scheduler and trying to do "tricks", I implemented a full task controller using celery, with tasks configurable from the admin panel.

Take note that, for now, if you want to enable/disable a task, you need to restart the container.

**Comment by terry-sydaus** [comment]
> Fixed on commit #[9b51149](https://github.com/raskitoma/pls_admin/commit/9b5114982072e606e2cbf5e20727015c1ec1590f)
> 
> Instead of relying on scheduler and trying to do "tricks", I implemented a full task controller using celery, with tasks configurable from the admin panel.
> 
> Take note that, for now, if you want to enable/disable a task, you need to restart the container.

Awesome work. The only celery I know about is the one I find in my stew ;-)

I am happy running with the scheduler for now. The adminpls docker container restarts on reboot and I set a cron job to run a shell script that contains the python3 -m flask scheduler command at reboot.

**Comment by raskitoma** [comment]
You'll need to disable the "scheduler" command, because it have been removed(deprecated).  I removed a library and replace it by `celery`, 'cause for me it's a better choice for task scheduling and deferring.

Pull the new version, run the "scheduler reset" command (it's on the README.md) and enter the admin panel and fill the 3 tasks needed for now (Price update, Validator Update, Wallet Update). Once filled that info on the admin panel, just restart the container again.

Also you will need to include a redis container (there's an example on the docker-compose.sample.yml) and the config needs to be update with those parameters as well (config file sample is updated as well)
